infrastructure/private-net-template.json

```diff
@@ -0,0 +1,28 @@
+{
+	"Genesis": {
+		"NetworkName": "devnet",
```

Comment:
shouldn't it be privatenet?

Reply:
I just moved the `infrastructure` folder to here as it is. It was same there so I didn't change here.